src/main/scala/mesosphere/marathon/DeprecatedFeatures.scala

```diff
@@ -53,7 +60,7 @@ object DeprecatedFeatures {
     softRemoveVersion = SemVer(1, 7, 0),
     hardRemoveVersion = SemVer(1, 8, 0))
 
-  def all = Seq(syncProxy, jsonSchemasResource, apiHeavyEvents, proxyEvents, kamonMetrics)
+  def all = Seq(syncProxy, jsonSchemasResource, apiHeavyEvents, proxyEvents, kamonMetrics, appC)
```

Comment:
Guys, I don't quite understand - here is a new `appC` flag that is not used anywhere? /cc @kensipe @timcharper  
Also, why the rush to land it? What happened to good-old-24h guideline?

Reply:
Yeah, we need to remove the appc flag :) Sorry, I thought this was clear in our conversation yesterday @kensipe :)

Reply:
I'd rather make use of it so that submitting AppC containers is only possible with `appC` flag active (which is inactive by default). This way we may even find out if anybody is actually using it.